daemon/src/projection.rs

```diff
@@ -0,0 +1,49 @@
+use crate::bitmex_price_feed::Quote;
+use crate::{Cfd, Order, UpdateCfdProposals};
+use tokio::sync::watch;
+use xtra_productivity::xtra_productivity;
+
+pub struct Actor {
+    tx_cfds: watch::Sender<Vec<Cfd>>,
```

Comment:
what does the `tx` here stand for?

Reply:
> Is it still a draft or is it ready to go in?

Sorry, it's ready for review and ready to ship from my perspective

Reply:
`tx` is the counterpart to `rx`.
- `tx` - used for the _transmitting_ side
- `rx` - used for the _receiving_ side